tests/manager/integration/patron_auth/test_sirsidynix_auth_provider.py

```diff
@@ -346,6 +346,79 @@ def test_remote_patron_lookup_bad_patron_read_data(
         assert patrondata.block_reason == SirsiBlockReasons.PATRON_BLOCKED
         assert patrondata.library_identifier == "testblocked"
 
+    @pytest.mark.parametrize(
+        "standing_key, approved, expected_blocked",
+        [
+            ("OK", True, False),  # Both approved AND standing=ok → not blocked
+            (
+                "ok",
+                True,
+                False,
+            ),  # Case insensitive: standing=ok (lowercase) → not blocked
```

Comment:
FYI - we could use `pytest.param` with an `id` here, which would make test failures easier to read:

```suggestion
            pytest.param("OK", True, False, id="both-not-blocked"),
            pytest.param(
                "ok",
                True,
                False,
                id="case-insensitive-standing-ok-not-blocked
            ),
```